src/components/v5/shared/RichText/RichText.tsx

```diff
@@ -52,7 +46,23 @@ const RichText: FC<RichTextProps> = ({
           </div>
         </>
       ) : (
-        notFormattedContent
+        <>
+          <span
+            className={clsx(
+              characterCount >= MIN_ANNOTATION_NUM,
```

Comment:
It will add `true` or `false` class to this span. Please remove it.

Reply:
fixed